src/app/services/mocks/constraint.service.mock.ts

```diff
@@ -93,4 +97,20 @@ export class ConstraintServiceMock {
     this._maxNumSearchResults = value;
   }
 
+  get validDimensionsUpdated(): Subject<Cohort[]> {
+    return this._validDimensionsUpdated;
+  }
+
+  set validDimensionsUpdated(value: Subject<Cohort[]>) {
+    this._validDimensionsUpdated = value;
+  }
+
+  get validDimensions(): Dimension[] {
```

Comment:
What are valid dimensions?

Reply:
all the dimensions that can be selected in the dropdown, when creating a constraint. I followed the naming used for pedigree constraint (`validPedigreeTypes`). I also thought about renaming this to `constraintDimensions` or something like this, but if you have other suggestions, please let me know.

Reply:
when people see `validDimensions`, they think there might be some invalid ones as well, how can I get the full list of dimensions?